### ISSUE TITLE
make-plural-string not working

### ISSUE THREAD

**Issue by demilsson** [danrevah/ngx-pipes#145]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
```

* **Ngx-pipes version:** 2.3.10

**Current behavior**
Using the pipe `make-plural-string` gives the error `Parser Error: Unexpected token '-'`.

Is dashes permitted in a pipe name? I believe it needs to be a valid JS identifier. If so, I believe the pipe `a-or-an` would also be affected. From a naming consistency point of view, `makePluralString` and `aOrAn` would be more aligned with the rest of the pipes. Actually, `pluralize` would be even more aligned.

**Expected behavior**
No error.

**Minimal reproduction of the problem with instructions**
```
{{ 'word' | make-plural-string }}
```

* **Angular version:** 7.1.2

* **Node (for AoT issues):** `node --version` =  v8.9.4

**Comment by danrevah** [comment]
Want to fix this?

**Comment by danrevah** [comment]
First, thank you for letting me know :)

I've published a fix to `v2.4.2`.
